app/models/github_organization.rb

```diff
@@ -98,6 +98,24 @@ def team_invitations_url
     "https://github.com/orgs/#{login}/invitations/new"
   end
 
+  # Public
+  #
```

Comment:
✂️  in [d5902e2](https://github.com/education/classroom/pull/668/commits/d5902e23672fb8d8ceeb40b85a57566b16c7e092)